src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -36,16 +36,38 @@ public void ConstructorNullName()
         public void ConstructorPathName()
         {
             string attachFile = Path.GetTempFileName();
-            try
+            using(var tempFile = new TempFile(attachFile))
```

Comment:
nit: space after `using`
optionally you can use `TempFile.Create(new byte[0])` then you can use TempFile.Path to access the path